semantic-ui/src/main/scala/react/semanticui/addons/textarea/TextArea.scala

```diff
@@ -70,7 +70,7 @@ object TextArea {
     var onInput: js.UndefOr[RawEvent] = js.native
 
     /** Indicates row count for a TextArea. */
-    var rows: js.UndefOr[Double | String] = js.native
+    var rows: js.UndefOr[Int | String] = js.native
```

Comment:
So, actually the fact that the compiler started complaining about these suggests to me that this macro is no longer `Unchecked` but is actually enforcing type-safety?